### DISCUSSION TITLE
How Does Drive Offsets Affect Ripping

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#30] [Announcements]
# How Does Drive Offsets Affect Ripping

If you are an experienced CD ripper, you may notice a thing called "offset". So how does it affect our ripping? I will reveal that to you.

https://zexwoo.blog/en/posts/research/offsets-affect-ripping/

**Comment by jgoulet921** [comment]
I just found out about your blog and it's great. Very informative articles, I hope you continue.
